### PR TITLE
feat: add entry for mergetool

### LINKED ISSUE #4096: [Entry] Subject: git-mergetool
### Reason/inspiration (optional)

I want to add this entry to the Git section because merge conflicts is something developers or people who code often encounter, so having this information would help them.  

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [ ] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by CBID2**
### Description
This PR adds an entry about the `mergetool` command in the Git section. I decided to create it because it would help developers and other tech professionals solve merge conflicts, a common encounter when working with code. 

### Type of Change

- Adding a new entry

### Issues
Closes #4096 

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/git/concepts/mergetool/mergetool.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+Title: 'mergetool' 
+Description: 'The mergetool command is used to activate a tool used to solve merge conflicts that occur in a branch'
```
Comment (dakshdeepHERE) [inline content/git/concepts/mergetool/mergetool.md:3]:
```suggestion
Description: 'The mergetool command is used to activate a tool used to solve merge conflicts that occur in a branch.' 
```

File: content/git/concepts/mergetool/mergetool.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Syntax
+
+```pseudo
+git mergetool
+```
+
+## Example
+
+Below is an example on how to use `git mergetool`. The developer runs the command and the merge conflict appears.
```
Comment (dakshdeepHERE) [inline content/git/concepts/mergetool/mergetool.md:25]:
```suggestion
Below is an example on how to use `git mergetool`. The developer runs the command and the merge conflict appears:
```

File: content/git/concepts/mergetool/mergetool.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+<<<<<<< HEAD
+## Creating a Tutorial
+We highly recommend using Markdown to create your tutorial for our audio editor.
+=======
+If you need help, check out our templates.
+>>>>>>> UPDATING-THE-DOCS
+```
+
+-`<<<<<<< HEAD`: where the developer's changes appears.  
+-`====`: Divides the developer's changes from the changes in the other branch, followed by `>>>>>>> UPDATING-THE-STYLE-GUIDE`
```
Comment (dakshdeepHERE) [inline content/git/concepts/mergetool/mergetool.md:39]:
```suggestion
- `<<<<<<< HEAD`: where the developer's changes appear.  
- `====`: Divides the developer's changes from the changes in the other branch, followed by `>>>>>>> UPDATING-THE-STYLE-GUIDE`
```

File: content/git/concepts/mergetool/mergetool.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+git commit -m "docs: update information about tutorials"
+```
+
+## Other tools
+
+Here are a list of other tools that are used to solve merge conflicts:
+
+- [Kaleidoscope](https://kaleidoscope.app/): This is often used by developers to resolve merge conflicts due to its use of different highlighted colors and powerful features ranging from a Convert to Merge command, text filters that remove unimportant data, and a debugger.
+- [Visual Studio Code](https://code.visualstudio.com/docs/sourcecontrol/overview#_3way-merge-editor): This Integrated Development Environment (IDE) has a built-in 3-way merge editor, which displays the incoming changes to file and provides a button that enables the the changes to be accepted into the file. This allows developers to easily edit files and resolve conflicts.
+- [Sublime Merge](https://www.sublimemerge.com/): Tool comes in a paid and free version. It is deemed to have a great user interface and intuitively easy to use.
```
Comment (dakshdeepHERE) [inline content/git/concepts/mergetool/mergetool.md:53]:
```suggestion
- [**Kaleidoscope**](https://kaleidoscope.app/): Developers often use this to resolve merge conflicts due to its use of different highlighted colors and powerful features ranging from a Convert to Merge command, text filters that remove unimportant data, and a debugger.
- [**Visual Studio Code**](https://code.visualstudio.com/docs/sourcecontrol/overview#_3way-merge-editor): This Integrated Development Environment (IDE) has a built-in 3-way merge editor, which displays the incoming changes to the file and provides a button that enables the changes to be accepted into the file. This allows developers to edit files and resolve conflicts easily.
- [**Sublime Merge**](https://www.sublimemerge.com/): Tool comes in a paid and free version. It is deemed to have a great user interface and is intuitively easy to use.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I am suggesting a few changes for you to work on @CBID2.

**Comment by CBID2** [comment]
Made the changes @dakshdeepHERE! :)

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

**Review by SSwiniarski — APPROVED** [review]
@CBID2 Looks good to me.

**Pull request merged**